custom/icds_reports/mobile_views.py

```diff
@@ -46,15 +46,15 @@ def password_reset(request, domain):
     kwargs['extra_context']['login_url'] = reverse('cas_mobile_dashboard_login', args=[domain])
     # so that we can redirect to a custom "done" page
     kwargs['post_reset_redirect'] = reverse('cas_mobile_dashboard_password_reset_done', args=[domain])
-    return auth_views.password_reset(request, **kwargs)
+    return auth_views.PasswordResetView.as_view(**kwargs)(request)
```

Comment:
fyi, this appears to have [broken password reset on the ICDS mobile dashboard](https://sentry.io/organizations/dimagi/issues/1589299416/). is it clear who should own fixing it? @dannyroberts

Reply:
nvm, took a quick look and it was simple enough: https://github.com/dimagi/commcare-hq/pull/27010